esp/esp/users/models/__init__.py

```diff
@@ -551,12 +551,6 @@ def addAvailableTime(self, program, timeslot, role=None):
         new_availability, created = UserAvailability.objects.get_or_create(user=self, event=timeslot, role=role)
         new_availability.save()
         
-    def convertAvailability(self):
-        resources = Resource.objects.filter(user=self)
-        for res in resources:
-            self.addAvailableTime(Program.objects.all()[0], res.event)
-        resources.delete()
```

Comment:
Appears to have been intended for use to convert from the old availibility system.  Presumably this is no longer necessary; it certainly has no callers.